.github/workflows/breaking-change-code.yaml

```diff
@@ -34,6 +34,8 @@ jobs:
             --tb "${{ github.event.pull_request.base.ref }}" \
             --hc "${{ github.event.pull_request.head.sha }}" \
             --repo "${{ github.repository }}"
+        env:
+          NODE_NO_WARNINGS: 1
```

Comment:
what are the node warnings?

Reply:
Node 22 reported [this warning] which blocks the openapi-diff run. The violation is in the autorest code, I think we might not want to fix it.
```
[DEP0060] DeprecationWarning: The util._extend API is deprecated. Please use Object.assign() instead.\n(Use node --trace-deprecation ... to show where the warning was created)
```